README.md

```diff
@@ -32,6 +32,11 @@ This is to give you, the reader, some sense of what we want to create:
 
 # Usage
 
+Prerequisites:
+```
+sudo apt-get install gcc-arm-none-eabi gcc-arm-linux-gnueabi
```

Comment:
We really only need the `arm-none-eabi` since we're only compiling the kernel and u-boot with it. The Makefile needs to be updated to reflect this however.